feature-libs/customer-ticketing/root/model/customer-ticketing.model.ts

```diff
@@ -1,3 +1,5 @@
+import { SortModel, PaginationModel } from '@spartacus/core';
+
 export const enum CUSTOM_CLASS {
   CLOSE = 'cx-text-gray',
   OPEN = 'cx-text-green',
```

Comment:
```suggestion
export const enum TEXT_COLOR_CLASS {
  INACTIVE = 'cx-text-gray',
  ACTIVE = 'cx-text-green',
  DEFAULT = '';
```

Reply:
I used Grey and Green because I don't want to add new variable names and assign them a color